app/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -41,4 +41,22 @@ class EnrolmentStoreProxyConnector @Inject()(appConfig: FrontendAppConfig, http:
         Logger.error("Enrolment Store Proxy error", exception)
         false
     }
+
+  lazy val serviceUrl = s"${appConfig.enrolmentStoreHost}/enrolment-store-proxy/enrolment-store"
+
+  def queryKnownFacts(utr: SAUTR, knownFacts: KnownFacts)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Boolean] = {
+    val knownFactsCombined = KnownFactsAndIdentifiers(utr.value, knownFacts.nino, knownFacts.postcode)
+
+    http.POST[KnownFactsAndIdentifiers, HttpResponse](s"$serviceUrl/enrolments", knownFactsCombined).map {
+        _.status match {
+          case OK         => true
+          case NO_CONTENT => false
+          case _          => false
+        }
```

Comment:
Could be simplified to `_.status == OK` if you're interested

Reply:
Good Spot ill get that changed